frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/emoji_picker/emoji_menu_item.dart

```diff
@@ -82,7 +82,7 @@ class _EmojiSelectionMenuState extends State<EmojiSelectionMenu> {
     if (event.logicalKey == LogicalKeyboardKey.escape &&
         event is KeyDownEvent) {
       //triggers on esc
-      widget.onExit();
+      widget.onExit?.call();
```

Comment:
Nice, I didn't even know about this function!

Reply:
It's pretty cool, it's based on this https://dart.dev/language/callable-objects